lib/sidekiq_rx_impl.cc

```diff
@@ -53,13 +53,19 @@ static const double STATUS_UPDATE_RATE_SECONDS{1.0};
 //#define DEBUG_1PPS_TIMESTAMP (1)
 
 sidekiq_rx::sptr sidekiq_rx::make(
+        int _card,
+        int port_id,
```

Comment:
If I'm understanding this correctly, this is really only supporting dual channels...so A1/A2.  This means that it can't be used for X4 to receive all 4 channels, correct?

Reply:
correct.  4 channels would be a bigger change.